### ISSUE TITLE
ARGOCD Not accessible from DNS after deployment with terraform

### ISSUE THREAD

**Issue by twintag-ciprian-pista** [awslabs/crossplane-on-eks#157]
ARGO CD is not available to access it from DNS URL 

#### Provide link to the example related to the quesiton
k8s-argocd-argocdar-b58ba5914a-1e2b2273bd87b3f1.elb.eu-central-1.amazonaws.com

Can anyone help?

**Comment by twintag-ciprian-pista** [comment]
Hello,

I am trying to access the LB of the ARGO cd to login in the Interface of
ARGO:
k8s-argocd-argocdar-b58ba5914a-1e2b2273bd87b3f1.elb.eu-central-1.amazonaws.com


This link is returning a timeout response. I already forwarded the port
locally and ARGOCD is working fine, I was able to access it, I checked the
RT IGW NATGW everything looks ok.

I do not know I am missing in order to access the ARGOCD UI.

Thanks
Ciprian
